content/docs/minutes/steering-meeting/_index.md

```diff
@@ -0,0 +1,20 @@
+# Steering meeting#
```

Comment:
I think it would be preferable to move all of the triage and steering meeting notes to `content/docs/minutes/{steering,triage}-YYYY-MM-DD.md` and have them all listed on the sidebar under "Minutes".